### ISSUE TITLE
Should negative tabindex fail in aria-hidden?

### ISSUE THREAD

**Issue by WilcoFiers** [act-rules/act-rules.github.io#1807]
This example was recently added to the `Element with aria-hidden has no focusable content` rule:

https://act-rules.github.io/rules/6cfa84#failed-example-5

```html
<div aria-hidden="true">
	<button tabindex="-1">Some button</button>
</div>
```

I didn't realise this when it went in, but looking at implementing this I can't quite figure out why this would fail SC 4.1.2. This is not a button in any sort of meaningful way. The only way I can see to get to it is with VoiceOver in reading mode. But it then announces the text node, not the button. It's explicitly called "some button text".

I think the expectation needs to be adjusted to only apply to elements in sequential focus navigation that do not redirect focus.

**Comment by straker** [comment]
The pattern to put `aria-hidden` on a parent element and `tabindex=-1` on all focusable descendant elements is a common pattern for hiding main content when a modal is open. The `inert` polyfill relies on this pattern. If the test case shown is truly a fail in terms of accessibility, is there an alternative accessible pattern to doing this?
